src/json/load_config.c

```diff
@@ -480,6 +481,62 @@ static wolfsentry_errcode_t convert_default_policy(JSON_TYPE type, const unsigne
 }
 
 static wolfsentry_errcode_t handle_eventconfig_clause(struct wolfsentry_json_process_state *jps, JSON_TYPE type, const unsigned char *data, size_t data_size, struct wolfsentry_eventconfig *eventconfig) {
+    if (jps->cur_depth == 5) {
```

Comment:
why at depth 5 for this case?

Reply:
that's just where they are in the JSON tree.  the `[` (open-array) pushes it down an extra level.